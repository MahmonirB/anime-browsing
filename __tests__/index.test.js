import renderer from 'react-test-renderer';

it('changes the class when hovered', () => {
  const component = renderer.create(<a href="http://www.facebook.com">Facebook</a>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
