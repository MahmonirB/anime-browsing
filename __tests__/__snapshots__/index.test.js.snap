// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<a
  href="http://www.facebook.com"
>
  Facebook
</a>
`;
